README.md

```diff
@@ -18,12 +18,12 @@ Anitab-Forms is an application that simplifies the processing and selection proc
 Documentation for the project is hosted [here](https://osp-web-docs.surge.sh/). We use `Docusaurus` for maintaining the documentation of the project.
 
 ### Few Useful Links:
-1. **`anitab-org wiki`** link - https://github.com/anitab-org/anitab-forms-web/wiki
-2. **`Open Sessions`** link - https://meet.google.com/eqb-nuut-kqm
-3. **`GSoC`** youtube video link  -  https://youtu.be/3A746GppZ0Y
-4. **`Setting up the project`** youtube video link -  https://youtu.be/_b2RQGbYN9w
-5. **`Question in Zulip chats`** link - https://anitab-org.zulipchat.com/#narrow/stream/223070-questions
-6. **`Announcement in Zulip chats`** link - https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements
+1. **anitab-org wiki** : [Visit](https://github.com/anitab-org/anitab-forms-web/wiki)
+2. **Open Sessions** : [Visit](https://meet.google.com/eqb-nuut-kqm)
+3. **GSoC** youtube video link : [Visit](https://youtu.be/3A746GppZ0Y)
+4. **Setting up the project** youtube video link : [Visit](https://youtu.be/_b2RQGbYN9w)
+5. **Question in Zulip chats** link : [Visit](https://anitab-org.zulipchat.com/#narrow/stream/223070-questions)
+6. **Announcement in Zulip chats** link : [Visit](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements)
```

Comment:
```suggestion
6. [Announcement in Zulip chats](https://anitab-org.zulipchat.com/#narrow/stream/213491-announcements): In this stream, all the important decision are announced by admins related to anitab-org opensource.
```
What do you think @Aaishpra about this representation i.e. link followed by some detail? This can be done for all the links.

Reply:
Can I do this changes?

Reply:
This is a good idea but I am still not sure about adding announcement and questions stream links in the readme as they are
common streams and not specifically dedicated to this project. Just adding links to our project stream should be fine. 🤔 What do you think?

Reply:
Although we can definitely add details for other streams.
@StarTrooper08 can you add these details by yourself or would you like some help?

Reply:
> This is a good idea but I am still not sure about adding announcement and questions stream links in the readme as they are common streams and not specifically dedicated to this project. Just adding links to our project stream should be fine. 🤔 What do you think?

I have one idea ,
we can just add one zulip chat link instead of those 2 channels links of zulip chat.


What do you say??

Reply:
Ya,  and the link should take you to the anitab-forms stream.(although I think we already have one in the badge).
Let's see what @codesankalp says

Reply:
> Although we can definitely add details for other streams.
> @StarTrooper08 can you add these details by yourself or would you like some help? 

Ok I will first try of my own and if stuck somewhere I will take help .

Reply:
> Ya,  and the link should take you to the anitab-forms stream.(although I think we already have one in the badge).
> Let's see what @codesankalp says

Ok